docs/features/custom-license-and-keyword-search-regular-expression-syntax.md

```diff
@@ -0,0 +1,285 @@
+<!--
```

Comment:
Is it worth just linking to this rather than copying the text?

Reply:
There are two reasons why we're copying rather than just linking

1) We want to be able to link directly to something that we own 
2) I made some minor edits to the file to remove the example Rust code that would probably just be confusing for most people reading this

Reply:
That makes sense, thanks!